README.rst

```diff
@@ -112,10 +112,11 @@ Running
 Tox will run unit and integration tests in both Python environments using a
 temporarily created virtualenv.
 
-You should ensure :code:`DCOS_CONFIG` is set and that the config file points
-to the Marathon instance you want to use for integration tests. If you're
-happy to use the default test configuration which assumes there is a Marathon
-instance running on localhost, set :code:`DCOS_CONFIG` as follows::
+You can set :code:`DCOS_CONFIG` to a config file that points to the
+Marathon instance you want to use for integration tests. If you're
```

Comment:
While we're updating these docs, perhaps you can change this to say "DCOS cluster" instead of "Marathon instance"?